### DISCUSSION TITLE
Exporting data

### DISCUSSION THREAD

**Discussion by CerielRoland** [cyring/CoreFreq#440] [Q&A] [answered]
Hi there,

Is there a way to export the data from corefreq to external systems?
Since the proxmox kernel does not support lm-sensors for my ryzen system I found this solution with all the data (and even more) i want to collect from the system. Is there a way to export it to use it in notification systems and dashboarding not being a terminal window?

Thanks!

Kind regards,
Ceriel

**Comment by cyring** [comment, marked as answer]
> Is there a way to export the data from corefreq to external systems?

Can a JSON export help ?
If so then use this argument:
```
corefreq-cli -j
```

**Reply by CerielRoland** [reply]
OMG how did I not see this in the manual....
